### DISCUSSION TITLE
关于WEB界面的一点建议

### DISCUSSION THREAD

**Discussion by shmily91** [wenda-LLM/wenda#276] [General]
每次输入时想换行，但是换不了，能否设置成按shift+enter进行换行

**Comment by ChenMo2017** [comment]
如果模型允许输入中包含换行符，我觉得以control+enter作为发送快捷键似乎更合适，但是我怀疑模型的输入是否允许包含回车换行（\n）。。。

**Comment by l15y** [comment]
现在支持按shift+enter
